app/models/state_file_az_intake.rb

```diff
@@ -147,6 +147,8 @@ def disqualifying_eligibility_rules
   end
 
   def disqualified_from_excise_credit_df?
+    return false if raw_direct_file_data.blank?
```

Comment:
is this a fix for something else? would have been nice to split this off into its own story

Reply:
Sorry - yes, there was a time when this PR called `show?` on _every_ controller in the flow, rather than just until it finds one that can be shown. That surfaced that we have a bunch of controllers whose `show?` methods crash when intakes are in certain states (typically when there's no DF data yet). I changed how `show?` is called, so we no longer hit these crashes (because I didn't want to also fix them across GYR & CTC) but I left these fixes in since we _shouldn't_ crash on any input to these methods